### PR TITLE
Various testing updates

### PR DESCRIPTION
**Pull request by kalisp**
## Brief description
Contains various updates for automatic testing.

Contains first attempt at dev documentation for testing.

Added new `Dev Docs` section to documentation, but couldn't figure out how to handle sidebar.js correctly. I would like to have  some pages (dev_requirements) in both Admin and Dev docs, if it is possible.

It is ready for first iteration of comments/requirements etc.

### CODE REVIEW

**Comment by mkolar** [comment]
Regarding the pages in sidebars, we'll have to do a few tests on what is possible in docusaurus, I believe we can show the same page in multiple places.

File: website/sidebars.js [hunk 1]
```diff
@@ -133,4 +132,17 @@ module.exports = {
             ],
         },
     ],
+     Dev: [
+        "dev_introduction",
+        {
+            type: "category",
+            label: "Dev documentation",
```
[the remark this answers is not in the document]
Comment (kalisp) [reply website/sidebars.js]: Flattened category,
Tried this https://docusaurus.io/docs/sidebar/multiple-sidebars to show Dev sidebar for `dev_build` and `dev_requirements`, didn't work, so moved those back to `Admin` section as they are important for Studio admins to build their own OP.
Added links to `dev_introduction` as it is pretty sparse right now.

**Comment by mkolar** [comment]
Hmm. we can have the same page under two sidebars, but it will only technically live under one of them anyways. I'm thinking that at the end of the day, `requirements` and `build` pages should be in the `dev docs` primarily. We should like to them from the Admin docs, but they are ultimately developer pages. 

My thinking is that Admin should be aimed mostly at studio admins and supervisors that deploy pre-build binaries and zips and mostly deal with configurations. While `dev` should be about the step before deployment.

**Comment by mkolar** [comment]
Actually I think let's merge this and open a conversation about the documentation structure and what's where in another issue.

**Comment by kalisp** [comment]
I was thinking the same about `requirements` and `build`, that it should be in Dev section. We could move them there, but should introduce new page to Admin section with where to get builds, with links do Dev section in case you would like to build it yourself.

**Review by mkolar — APPROVED** [review]

**Pull request merged**